hanadb_exporter/exporters/prometheus_exporter.py

```diff
@@ -94,15 +66,16 @@ def collect(self):
 
         for query in self._metrics_config.queries:
             #  execute each query once (only if enabled)
-            if query.enabled:
+            if query.enabled and self._hana_version >= query.hana_version:
```

Comment:
We need to consider a minimal and maximum version of execution.  There are tables that have columns included on newer SP, that we could maintain different versions of the same query. On that is executed up 2.00.030 for example, and another that is executed from 2.00.040.

Reply:
Ok, this was not considered. To fix that I would need to include some changes

hana_version being a list maybe, 
- if there are 2 values, this would be a range
- having only 1 value, would consider beyond that version

@diegoakechi What do you think?

Reply:
@arbulu89 Sounds good. Maybe the param should be named like `hana_version_range` or something like that.